http/obmc_shell.hpp

```diff
@@ -179,6 +179,15 @@ inline void requestRoutes(App& app)
         .onopen([](crow::websocket::Connection& conn,
                    const std::shared_ptr<bmcweb::AsyncResp>&) {
             BMCWEB_LOG_DEBUG << "Connection " << &conn << " opened";
+
+            // TODO: this user check must be removed when WebSocket privileges
+            // work.
+            if (conn.getUserName() != "service")
+            {
+                BMCWEB_LOG_DEBUG
+                    << "only service user have access to obmc_shell";
+                conn.close("only service user have access to bmc console");
```

Comment:
Do we need a return code here ? A 403 ?

Reply:
we can, but that makes this more complicated.
because code reaches here because we accept the HTTP requests and upgrade (handle upgrade (convert HTTP to WSS)) also completed. that mean user already gets 101 (upgrade ok) for it. after that, sending 403 might be strange.

Reply:
Ack.